<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>App</class>
 <widget class="QWidget" name="App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>311</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_connect">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_address">
      <property name="text">
       <string>127.0.0.1:16384</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>391</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_open_stream">
      <property name="text">
       <string>开启推流</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_close_stream">
      <property name="text">
       <string>关闭推流</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>最大帧率限制：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="input_max_rate">
      <property name="maximum">
       <number>240</number>
      </property>
      <property name="value">
       <number>120</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="VideoWidget" name="video_display" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>1021</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>render/videowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
